update attribute controls for xpress-9.4

mbataillou opened the pull request:
Ran the `build_param_control_dicts.jl` script and kept old parameters for compatibility.

Closes #272

odow commented on src/attributes_controls.jl:
```diff
@@ -1,27 +1,22 @@
-#
-# Use of this source code is governed by an MIT-style license that can be found
-# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
-
```
Please keep

odow commented on src/attributes_controls.jl:
```diff
@@ -1,27 +1,22 @@
 =#
 
-const STRING_CONTROLS = Dict{String,Int32}(
+const STRING_CONTROLS = Dict{String, Int32}(
```
I think the formatter is going to complain about this

odow reviewed (commented):
We also need to update the constants at the bottom of https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl

mbataillou reviewed (commented):
Added the license back and fixed format.

mbataillou commented:
I am not sure how to update the constants in  https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl. Where do I find the new values, the ones there do not seem to map to the attribute controls.

mbataillou commented:
> Added the license back and fixed format.



> We also need to update the constants at the bottom of https://github.com/jump-dev/Xpress.jl/blob/master/src/Lib/xprs.jl

Updated that in the last commit. Tell me if I missed anything!

odow reviewed (approved):
LGTM
